layouts/shortcodes/aas-workflow-linux.md

```diff
@@ -0,0 +1,3 @@
+To update your Datadog instrumentation with zero downtime, Datadog provides a [GitHub workflow][workflow].
```

Comment:
Can we be more explicit about the fact we are only providing examples using Azure's github actions. I want customers to be clear that these tools are owned by Azure and not us. I'm concerned that by saying we provide the workflow, it will lead to more escalations.